tests/operations.go

```diff
@@ -43,6 +50,9 @@ type Operations struct {
 	PVCClient      *pvc.Kubeclient
 	NSClient       *ns.Kubeclient
 	SnapClient     *snap.Kubeclient
+	CSPClient      *csp.Kubeclient
+	SPCClient      *spc.Kubeclient
+	SVCClient      *svc.Kubeclient
```

Comment:
IMO if this struct will keep on adding new fields we are breaking `Open Closed Principle`